Library/ViewModels/LiveStreamDiscoveryViewModel.swift

```diff
@@ -13,6 +16,9 @@ public protocol LiveStreamDiscoveryViewModelInputs {
 
   /// Call when the view loads.
   func viewDidLoad()
+
+  /// Call when the viewWillAppear
+  func viewWillAppear()
```

Comment:
Nope, I tried doing it with just that - turns out it's [not](http://stackoverflow.com/questions/5277940/why-does-viewwillappear-not-get-called-when-an-app-comes-back-from-the-backgroun) called.